listen.go

```diff
@@ -50,6 +50,11 @@ type ListenConfig struct {
 	// Default: nil
 	TLSConfigFunc func(tlsConfig *tls.Config) `json:"tls_config_func"`
 
+	// TLSMinVersion allows to set tls minimum version.
+	//
+	// Default: VersionTLS12
+	TLSMinVersion uint16 `json:"tls_min_version"`
```

Comment:
Is `tls.Version12` not a type?

Reply:
No, it's not. crypto/tls use uint16 for configuring tls.
https://pkg.go.dev/crypto/tls#Config.MinVersion

We can create a type for TLS versions in Fiber, but this is not the best practice.

Reply:
Ok, cool